src/js/dp/timeseries.js

```diff
@@ -0,0 +1,266 @@
+const timeSeriesContainer = document.querySelector("#timeSeriesContainer");
+//time series
+if(timeSeriesContainer){
+    var listContainer = timeSeriesContainer.querySelector(".timeseries__list-container"),
+        basket = timeSeriesContainer.querySelector("#timeseries__basket"),
+        resultsContainer = timeSeriesContainer.querySelector("#results"),
+        buttons = timeSeriesContainer.querySelectorAll(".timeseries__remember, .timeseries__download"),
+        noTimeseries = timeSeriesContainer.querySelector(".timeseries__empty"),
+        xlsForm = timeSeriesContainer.querySelector("#xls-form"),
+        csvForm = timeSeriesContainer.querySelector("#csv-form"),
+        list = timeSeriesContainer.querySelector(".timeseries__list"),
+        timeseriesList = {},
+        basketCookieName = "timeseriesBasket",
+        rememberCookieName = "rememberBasket",
+        listCount = 0,
+        counter = timeSeriesContainer.querySelector("#timeseries__count"),
+        exitBtn = timeSeriesContainer.querySelector(".timeseries__list--exit"),
+        rememberCb = timeSeriesContainer.querySelector("#remember-selection");
+        
+    exitBtn.addEventListener("click", () => {
+        boxWithStuff.classList.toggle("hidden");
+    });
+
+    function initialize() {
+        remember = getLocalStorageProperty(rememberCookieName);
+        if (remember) { 
```

Comment:
Do we want/need two `if (remember)` straight after each other (line 26 & 33)? Any reason the logic couldn't be in the same block?

Reply:
Apologies, it was supposed to check if remember is defined on line 26. Fixed now!